sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -1,12 +1,13 @@
 # Azure Storage Files Data Lake client library for .NET
 
 > Server Version: 2019-02-02
+
 Azure Data Lake includes all the capabilities required to make it easy for developers, data scientists, 
 and analysts to store data of any size, shape, and speed, and do all types of processing and analytics 
 across platforms and languages. It removes the complexities of ingesting and storing all of your data
 while making it faster to get up and running with batch, streaming, and interactive analytics.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
```

Comment:
Why are we removing the NuGet package link?

Reply:
We haven't released the nuget package in preview yet.

Reply:
Sure, but when you do release the URL will be https://www.nuget.org/packages/Azure.Storage.Files.DataLake so we might as well keep that in the README.  It'll be valid as soon as you publish.